documentation/modules/oauth/con-mapping-keycloak-authz-services-to-kafka-model.adoc

```diff
@@ -24,7 +23,7 @@ All permissions exist within the scope of the `kafka` client. If you have differ
 When the Kafka client uses OAUTHBEARER authentication, the Keycloak authorizer (`KeycloakAuthorizer`) uses the access token of the current session to retrieve a list of grants from the Keycloak server.
 To retrieve the grants, the authorizer evaluates the Keycloak Authorization Services policies and permissions.
```

Comment:
While not part of this set of changes this sentence is not correct. The way it reads it seems like the authorizer needs to know about Keycloak Authorization Services policies and permissions in order to retrieve grants. But actually, authorizer just sends the access token in an HTTP request, and gets back a list of grants (Kafka ACL permissions in the form of a string of a special format). The evaluation was performed on the Keycloak by Keycloak Authorization Services to produce that list of grants from policies and permission configured for the owner of the access token.

```suggestion
To grant permissions the authorizer evaluates the received (and locally cached) list of grants produced by Keycloak Authorization Services based on policies and permissions configured for the owner of the access token.
```